{{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml

```diff
@@ -0,0 +1,116 @@
+---
+dependency:
+  name: {{ cookiecutter.dependency_name }}
+  options:
+    role-file: requirements.yml
+driver:
+  name: ec2
+lint:
+  name: {{ cookiecutter.lint_name }}
+  options:
+    config-file: .yamllint
+platforms:
+  - name: test-aws-centos7-{{ cookiecutter.role_name }}
+    image: ami-9887c6e7
+    instance_type: m5.large
+    region: us-east-1
+    vpc_subnet_id: {{ cookiecutter.vpc_subnet_id }}
+    assign_public_ip: False
+    spot_price: 0.04
+    security_group_name:
+      - default
+    wait_timeout: 1800
+    ssh_user: centos
+    groups:
+      - rhel_family
+
+  - name: test-aws-centos6-{{ cookiecutter.role_name }}
+    image: ami-1585c46a
+    instance_type: m4.large
+    region: us-east-1
```

Comment:
should be replaced for using variables value from cookiecutter

Reply:
Done